android/quest/src/test/java/org/smartregister/fhircore/quest/ui/pdf/PdfLauncherFragmentTest.kt

```diff
@@ -85,7 +82,7 @@ class PdfLauncherFragmentTest : RobolectricTest() {
 
     activity.supportFragmentManager.beginTransaction().add(fragment, null).commitNow()
 
-    coVerify { pdfLauncherViewModel.retrieveQuestionnaireResponse(any(), any(), any()) }
+    coVerify { pdfLauncherViewModel.retrieveQuestionnaireResponse(any(), any()) }
     coVerify { pdfLauncherViewModel.retrieveBinary(any()) }
```

Comment:
Is it possible to update these tests to avoid using mock so you can test the functionality for `generatePdfWithHtml` and perform relevant assertions>?